python_module/cmake/add_python_export_library.cmake

```diff
@@ -12,9 +12,15 @@
 # Set the path for the output python files. This should be the path
 # with the __init__.py file. The standard for ROS (where python message
 # definitions live) is ${PROJECT_SOURCE_DIR}/src/${PROJECT_NAME}
+include(CMakeParseArguments)
```

Comment:
I'd swap this with the comment above because the comment is for the function not the include.